sympy/solvers/tests/test_solvers.py

```diff
@@ -1557,23 +1547,61 @@ def test_issue_14607():
         assert s[0][var].simplify() == knownsolution[var].simplify()
 
 
-@slow
 def test_lambert_multivariate():
     from sympy.abc import a, x, y
-    from sympy.solvers.bivariate import _filtered_gens, _lambert, _solve_lambert
-
+    from sympy.solvers.bivariate import _filtered_gens, _lambert
```

Comment:
`_filtered_gens` seems to be used again at L1581. Maybe you should import it (and others) at the top.

Reply:
Done